daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -303,6 +303,13 @@ function Install-32bitPackages {
   }
 }
 
+function Enable-WinRM {
+  if ($pn -like '*Enterprise') {
```

Comment:
Is this matching correct: "*Enterprise" means Windows Client? I would have expected it to be the opposite.

Reply:
Yes, Windows 7/8.1/10 Enterprise are the client version that is available via volume licence agreements. They started usinng the Enterprise edition SKU in vista.

A full list of the client and server product names is at https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions